src/styles/index.scss

```
@@ -1,3 +1,232 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+  * {
+    box-sizing: border-box;
+  }
+}
```

If you want box-sizing: border-box; applied to all elements, it must be done separately and not inside another selector.

Here is the corrected code:

```suggestion
* {
    box-sizing: border-box;
}

body {
  margin: 0;
  font-family: Roboto, sans-serif;
}
```